venues/thecvf.com/ECCV/2020/Conference/python/emergency_reviews.py

```diff
@@ -0,0 +1,170 @@
+import openreview
+import csv
+from tqdm import tqdm
+import argparse
+
+if __name__ == '__main__':
+    ## Argument handling
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--baseurl', help="base url")
+    parser.add_argument('--username')
+    parser.add_argument('--password')
+    args = parser.parse_args()
+
+    client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+    reviewer_group = client.get_group('thecvf.com/ECCV/2020/Conference/Reviewers')
+    emergency_reviewer_group = client.post_group(openreview.Group(
+        id = 'thecvf.com/ECCV/2020/Conference/Emergency_Reviewers',
+        signatures = ['thecvf.com/ECCV/2020/Conference'],
+        signatories = ['thecvf.com/ECCV/2020/Conference/Emergency_Reviewers'],
+        readers = [
+            'thecvf.com/ECCV/2020/Conference',
+            'thecvf.com/ECCV/2020/Conference/Area_Chairs',
+            'thecvf.com/ECCV/2020/Conference/Emergency_Reviewers'],
+        writers = ['thecvf.com/ECCV/2020/Conference',
+        'thecvf.com/ECCV/2020/Conference/Area_Chairs'],
+        members = []
+    ))
+    print ('Posted group:', emergency_reviewer_group.id)
+
+    emergency_load_invitation = client.get_invitation('thecvf.com/ECCV/2020/Conference/Reviewers/-/Custom_Load')
+    emergency_load_invitation.id = 'thecvf.com/ECCV/2020/Conference/Reviewers/-/Emergency_Load'
```

Comment:
why are you changing the id or a current invitation instead of creating a new one?